packages/components/src/composite/Tabs/Container.tsx

```
@@ -116,42 +125,52 @@
-            if (entry && entry.contentRect.height) {
-              (
-                listContainerRef.current as HTMLElement
-              ).style.maxHeight = `${entry.contentRect.height}px`;
+        if (height) {
+          (
+            listContainerRef.current as HTMLElement
+          ).style.maxHeight = `${height}px`;
+          setTimeout(() => {
+            resizeObserverRef.current = new ResizeObserver((entries) => {
+              const entry = entries[0];
+              if (entry && entry.contentRect.height) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)